bin/cloud-provision.sh

```diff
@@ -16,6 +16,53 @@ fi
 NOVARC_PATH=$1
 SPI_CREDENTIALS_PATH=$2
 OPENSTACK_CREDENTIALS_DIR="$JENKINS_HOME/.openstack"
+PRIVATE_KEY_PATH=~/.canonistack/${OS_USERNAME}_${OS_REGION_NAME}.key
```

Comment:
Why isn't this one an argument?
I have my personal keys, the team keys for canonistack and scalingstack. It would be nicer as an argument.